packages/tables/resources/views/columns/layout/panel.blade.php

```diff
@@ -1,7 +1,11 @@
-<div @class([
-    'px-4 py-3 bg-gray-100 rounded-lg',
-    'dark:bg-gray-900' => config('forms.dark_mode'),
-])>
+<div
+    {{ $attributes
+        ->merge($getExtraAttributes())
+        ->class([
+            'px-4 py-3 bg-gray-100 rounded-lg',
+            'dark:bg-gray-900' => config('forms.dark_mode'),
+        ]) }}
+>
```

Comment:
Same thing for the other files.

```suggestion
<div
    {{
        $attributes
            ->merge($getExtraAttributes())
            ->class([
                'px-4 py-3 bg-gray-100 rounded-lg',
                'dark:bg-gray-900' => config('forms.dark_mode'),
            ])
    }}
>
```